cfme/infrastructure/pxe.py

```diff
@@ -369,6 +372,7 @@ def exists(self):
         return self.name in [s.name for s in candidates]
 
     @exists.variant('ui')
+    @property
```

Comment:
It may not, like with the Sentaku instances. Our tests for pxe never get to calling it because rhevm36 infra isn't setup correctly for CFME to display the pxe servers